docs/layout/dynamic-font-scaling.md

```diff
@@ -0,0 +1,205 @@
+
+Also note that any Ionic components that have had their font sizes overridden should also be updated to use `rem` units.
+
+### Custom Font Family
+
+We recommend using the default fonts in Ionic as they are designed to look good at any size and ensure consistency with other mobile apps. However, developers can use a custom font family with Dynamic Font Scaling via CSS:
+
+```css
+html {
+  --ion-dynamic-font: var(--ion-default-dynamic-font);
+  --ion-font-family: 'Comic Sans';
+}
+```
+
+### `em` units versus `rem` units
+
+Developers have two options for relative font sizes: `em` and `rem`.
+
+`em` units set the font size of an element relative to the font size of its parent.
+
+In the following example, the computed font size of `.child` is `40px` because it is a child of `.parent` (2em \* 20px = 40px).
+
+```css
+.parent {
+  font-size: 20px;
+}
+
+.child {
+  font-size: 2em;
+}
+```
+
+However, the `em` unit has a compounding effect which can cause issues. In the following example, the second `.child` element has a computed font size of `80px` since the font sizes compound.
+
+```html
+<div class="parent">
+  Parent element with 20px
+  <div class="child">
+    Child element with 40px
+    <div class="child">Child element with 80px</div>
+  </div>
+</div>
+```
+
+<div style={{ fontSize: '20px' }}>
+  Parent element with 20px
+  <div style={{ fontSize: '2em' }}>
+    Child element with 40px
+    <div style={{ fontSize: '2em' }}>Child element with 80px</div>
+  </div>
+</div>
+
+Due to this compounding effect, we strongly recommend using `rem` units instead of `em` units when working with Dynamic Font Scaling. `rem` units set the font size of an element relative to the font size of the root element, which is typically `<html>`. The default font size of the root element is typically `16px`.
+
+In the following example, the computed font size of `.child` is `32px` because the font size is being computed relative to `html`, not `.parent`.
+
+```css
+.parent {
+  font-size: 20px;
+}
+
+.child {
+  font-size: 2rem;
+}
+```
+
+## How Dynamic Font Scaling works in Ionic
+
+All Ionic components that define font sizes use [rem units](https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/Values_and_units#lengths). This sizes the text in each component relative to the font size of the root element, which is usually the `html` element. This means that as the root element's font size changes, the text in all Ionic components scale in a consistent manner. This avoids the need to manually override each component's font size.
```

Comment:
> All Ionic components that define font sizes use rem units

This is inaccurate, no? We have some components that don't scale (tabs) and others that use `em` units (icons).

Reply:
That's true. Clarified in https://github.com/ionic-team/ionic-docs/pull/3111/commits/338b4730b0c039518c63e0d303af9ff17c4444b2